src/Http/Request.php

```diff
@@ -258,6 +275,9 @@ public function getParsedBody() {
    *   The parsed body.
    */
   protected static function parseBody($method) {
+    if (!static::isWriteMethod($method)) {
```

Comment:
Technically the body of the request is what comes after the headers in the http request.

In 7.x-1.x we have the `$request` variable that can hold the body and the query string input arguments. 7.x-2.x separates the two concepts.